### ISSUE TITLE
feat(ion-select): ion-select can apply changes without Okbutton click

### ISSUE THREAD

**Issue by vzharkov** [driftyco/ionic#10178]
**Ionic version:**  (check one with "x")
[ ] **1.x**
[ x] **2.x**

**I'm submitting a ...**  (check one with "x")
[ ] bug report
[ x] feature request
[ ] support request => Please do not submit support requests here, use one of these channels: https://forum.ionicframework.com/ or http://ionicworldwide.herokuapp.com/

**Current behavior:**
We need to click on "Okbutton" to check value for single ion-select.

**Expected behavior:**
Select and close Alert dialog for single ion-select just by click on needed ion-option item. In this case buttons are hidden. This may be optional flag to ion-select.
The behaviour for multiple ion-select does not change.

**Other information:**
We need this behaviour for our product. Also i found that the material design component have the same behaviour (https://material.angular.io/components/component/select). Also i saw that this feature already requested by other ionic developers (https://github.com/driftyco/ionic/issues/7150)

I can implement this feature by myself and make pull-request, if community and ionic team interested in this.

**Ionic info:** (run `ionic info` from a terminal/cmd prompt and paste output below):
```
Cordova CLI: 6.4.0 
Ionic Framework Version: 2.0.0
Ionic CLI Version: 2.2.1
Ionic App Lib Version: 2.2.0
Ionic App Scripts Version: 1.0.0
ios-deploy version: Not installed
ios-sim version: Not installed
OS: Linux 4.4
Node Version: v6.6.0
Xcode version: Not installed
```

**Comment by 64jcl** [comment]
Would love to have this feature. The current action sheet interface version isn't particularly good and limited to maximum 6 items. Also the current rendering of the select list even for single select has rather slow rendering of the bullets on a real Android device. This would avoided altogether if each select was a button in the popup.

**Comment by jgw96** [comment]
Thanks everyone! We will consider this feature request.

**Comment by jgw96** [comment]
Hello everyone! Thanks for the feature request. I'm going to move this issue over to our internal list of feature requests for evaluation. We are continually prioritizing all requests that we receive with outstanding issues. We are extremely grateful for your feedback, but it may not always be our next priority. I'll copy the issue back to this repository when we have begun implementation. Thanks!

**Comment by jgw96** [comment]
This issue was moved to driftyco/ionic-feature-requests#96

**Comment by vzharkov** [comment]
@jgw96 hi. How can i know a status of this feature request?

**Comment by darthranjith** [comment]
Is this feature implemented or not?

**Comment by Ross-Rawlins** [comment]
Is this live?

**Comment by jinisha** [comment]
Does anyone know when will this be implemented? Also, why no one from ionic team is responding to the above questions asked by users?

**Comment by 27holdings** [comment]
Really need this feature also.

**Comment by RichardM99** [comment]
+1

**Comment by ngehlert** [comment]
any updates? @jgw96 your link to the feature request does not work

**Comment by ivanthecrazy** [comment]
+1

**Comment by wormling** [comment]
+1

**Comment by Nephrited** [comment]
+1

**Comment by Nephrited** [comment]
I believe this has been implemented! Try the interface option "popover"

**Comment by ngehlert** [comment]
kind of true but popover looks completely different... :(

**Comment by ivanthecrazy** [comment]
Popover cannot be used of you have more than 6 items in select. In this case it will be automatically converted to alert.

**Comment by Nephrited** [comment]
@ivanthecrazy That doesn't appear to be the case. Using a popover successfully with about 80 items at the moment.

**Comment by ngehlert** [comment]
@ivanthecrazy @Nephrited he probably mixed it up with the action sheet interface. but still popover looks different and is smaller than the alert interface

**Comment by socca1157** [comment]
+1.......

**Comment by wormling** [comment]
+1

**Comment by peterpan666** [comment]
Ok guys I think I've got a workaround for this.

I've forked a plunker to show you, it doesn't work on Ionic 2 but works great on Ionic 4, though I don't know how to update the ionic version on plunker 😕

Look at the files `home.ts` and `home.html`.

- First I get the select with the ViewChild decorator.

- Then I listen for the (ionSelect) event on the ion-option and reassign the corresponding value.

- Every time the value updates, it closes the alert.

- BANG. Working.

Next you just have to had some style to remove the buttons if you want to.

https://embed.plnkr.co/d6Njw3hteUkCa0jq4zb4/
